release-notes/2.2/2.2-supported-os.md

```diff
@@ -15,6 +15,7 @@ OS                            | Version                       | Architectures  |
 Windows Client                | 7 SP1+, 8.1                   | x64, x86       |
 Windows 10 Client             | Version 1607+                 | x64, x86       |
 Windows Server                | 2008 R2 SP1+                  | x64, x86       |
+Nano Server                   | Version 1709+                 | x64, ARM32     |
```

Comment:
Should there be an * explaining that ARM32 support starts in 1809?

Reply:
In addition, can we have a line for IoTCore as well?